### PR TITLE
feat: mark strings for i18n on lpr page

### PR DESCRIPTION
**Pull request by muhammad-ammar**
# For all changes

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/Admin/index.jsx [hunk 1]
```diff
@@ -211,14 +270,23 @@ class Admin extends React.Component {
...
 
   renderDownloadButton() {
-    const { actionSlug } = this.props;
+    const { actionSlug, intl } = this.props;
     const tableMetadata = this.getMetadataForAction(actionSlug);
+    let downloadButtonLabel;
+    if (actionSlug) {
+      downloadButtonLabel = intl.formatMessage({
+        id: 'admin.portal.lpr.current.report.csv.download',
+        defaultMessage: 'Download current report (CSV)',
```
Comment (jajjibhai008) [inline src/components/Admin/index.jsx:279]: where we are handling `Download full report (CSV)` ?

**Review by jajjibhai008 — APPROVED** [review]
LGTM 👍 
[Verify] please do verify that `npm run i18n_extract` is working fine.

File: src/components/Admin/index.jsx (hunk 1 above)
Comment (muhammad-ammar) [reply src/components/Admin/index.jsx:279]: `Download full report (CSV)` is handled [here](https://github.com/openedx/frontend-app-admin-portal/blob/3fcadf5f1337b2f1cacebb1740735e9dff5b0498/src/components/DownloadCsvButton/index.jsx#L40)

**Comment by muhammad-ammar** [comment]
> npm run i18n_extract

executed `npm run i18n_extract` and it completed without any error.

**Pull request merged**